test/functional/rpc_net.py

```diff
@@ -340,26 +340,52 @@ def test_addpeeraddress(self):
         assert_raises_rpc_error(-1, "JSON integer out of range", self.nodes[0].addpeeraddress, address="1.2.3.4", port=-1)
         assert_raises_rpc_error(-1, "JSON integer out of range", self.nodes[0].addpeeraddress, address="1.2.3.4", port=65536)
 
+        self.log.debug("Test that adding a valid address to the new table succeeds")
+        assert_equal(node.addpeeraddress(address="1.0.0.0", tried=False, port=8333), {"success": True})
+        addrman = node.getrawaddrman()
+        assert_equal(len(list(addrman["tried"].values())), 0)
```

Comment:
edc8cb64: why not?
```suggestion
        assert_equal(len(addrman["tried"]), 0)
```
(left 1 more similar suggestion)